### PR TITLE
Events prototypes 2

### PR DESCRIPTION
**Pull request by gestchild**
## Who is this for?
The team

References #2944

## What is it doing for them?

- Creates Option 2 for testing
- Displays date range and number of events for a multi date event 
- Adds an anchor link for viewing all dates and booking lower down the page

jul-03-2018 16-50-25

### still to do 
- eventbrite widget integration

### CODE REVIEW

**Comment by jamesgorrie** [comment]
re: eventbrite integration - we have a way of making the iframe height change by proxying EB ([we did this way at the beginning of working on whats on](https://github.com/wellcometrust/wellcomecollection.org/pull/1149)) - but we can fix it for the test.

File: client/js/app.js [hunk 1]
```diff
@@ -36,6 +36,7 @@ import tabs from './components/tabs';
 import {eventbriteTicketStatus} from './components/eventbrite-ticket-status';
 import newsletterSignup from './components/newsletter-signup';
 import {createMaps} from './components/map';
+import {onWindowResizeDebounce$} from './utils/dom-events';
```
Comment (davidpmccormick) [inline client/js/app.js]: I was wondering about the cycle stuff yesterday – if/when we get rid of the sticky-digital-story-nav, should we think about getting rid of cycle? It's a fair chunk of code that we've added for very little gain. And we could use e.g. lodash for debounce/throttle.

Only mentioning here while I remember; not suggesting it should be anything to do with this PR.

File: common/views/components/BasePage/EventPageV2.js [hunk 2]
```diff
@@ -0,0 +1,292 @@
...
+    contentUrl: image.contentUrl,
+    title: image.title,
+    author: image.author,
+    sourceName: image.source && image.source.name,
+    sourceLink: image.source && image.source.link,
+    license: image.license,
+    copyrightHolder: image.copyright && image.copyright.holder,
+    copyrightLink: image.copyright && image.copyright.link
+  };
+  /* https://github.com/facebook/flow/issues/2405 */
```
Comment (davidpmccormick) [inline common/views/components/BasePage/EventPageV2.js]: Believe we can remove this FlowFixMe now

File: common/views/components/BasePage/EventPageV2.js [hunk 3]
```diff
@@ -0,0 +1,292 @@
...
+            return (
+              <div className='body-text' key={audience.title}>
+                <h2>For {audience.title}</h2>
+                <PrismicHtmlBlock html={audience.description} />
+              </div>
+            );
+          }
+        })}
+
+        {/* Booking CTAs */}
```
Comment (davidpmccormick) [inline common/views/components/BasePage/EventPageV2.js]: Does this need to be left in?

File: client/js/app.js (hunk 1 above)
Comment (jamesgorrie) [reply client/js/app.js]: 👍

**Pull request merged**